modules/ROOT/pages/source/payload-mode.adoc

```diff
@@ -0,0 +1,185 @@
+= Kafka Source Connector: Payload Mode Configuration
+
+The Kafka Source Connector for Neo4j supports two payload modes to control the format of data serialized and published to Kafka topics: `EXTENDED` and `COMPACT`. This feature is configurable through the `neo4j.payload-mode` property, allowing users to select the preferred serialization format based on data requirements.
+
+== Payload Modes
+
+The `neo4j.payload-mode` configuration offers the following options:
+
+* **`EXTENDED` (Default)**: Provides a detailed structure for each property, supporting schema compatibility and consistency. This format is especially useful in cases where schema changes (such as property type changes) or temporal types are present, ensuring data consistency across changes.
+
+* **`COMPACT`**: Produces a simpler format that only includes the essential fields. This format is lighter and may be preferable when schema compatibility or complex data types are not required.
+
+[WARNING]
+====
+*Limitations of COMPACT Mode*
+
+* **Property Type Changes**: COMPACT mode does not support changes in property types. If a property type changes in Neo4j (e.g., from integer to string), it can break the schema.
+* **Protobuf Compatibility**: COMPACT mode is not supported with Protobuf. It does not support serialization of temporal types (e.g., `LocalDate`, `LocalDateTime`).
+====
+
+
+== Configuration
+
+The payload mode can be configured in the source connector's settings as follows:
+
+[source,json]
+----
+"neo4j.payload-mode": "EXTENDED" // Or "COMPACT" based on requirements
+----
+
+== Example Data Formats
+
+The following examples show how data will be published in each payload mode.
+
+=== COMPACT Mode Example
```

Comment:
Should use backticks (monospace) for consistency. See the next paragraph too.

Reply:
You mean backsticks for all COMPACT and EXTENDED texts?

Reply:
👍